Libraries/src/Amazon.Lambda.RuntimeSupport/Helpers/FileDescriptorLogStream.cs

```diff
@@ -107,12 +110,14 @@ public override void Write(byte[] buffer, int offset, int count)
                 }
 
                 var typeAndLength = ArrayPool<byte>.Shared.Rent(LambdaTelemetryLogHeaderLength);
+                var now = (DateTimeOffset.UtcNow - UnixEpoch).Ticks / 10; // There are 10 tick per microsecond
                 try
                 {
                     Buffer.BlockCopy(_frameTypeBytes, 0, typeAndLength, 0, 4);
                     Buffer.BlockCopy(messageLengthBytes, 0, typeAndLength, 4, 4);
+                    Buffer.BlockCopy(now, 0, typeAndLength, 8, 8);
```

Comment:
This does not compile because `now` is a `long` but the `BlockCopy` method needs a `byte[]`. Below is how I changed the function to compile.

```
            public override void Write(byte[] buffer, int offset, int count)
            {
                var messageLengthBytes = BitConverter.GetBytes(count - offset);
                if (BitConverter.IsLittleEndian)
                {
                    Array.Reverse(messageLengthBytes);
                }

                var now = (DateTimeOffset.UtcNow - UnixEpoch).Ticks / 10; // There are 10 tick per microsecond
                var nowInBytes = BitConverter.GetBytes(now);
                if (BitConverter.IsLittleEndian)
                {
                    Array.Reverse(nowInBytes);
                }

                var typeAndLength = ArrayPool<byte>.Shared.Rent(LambdaTelemetryLogHeaderLength);
                try
                {
                    Buffer.BlockCopy(_frameTypeBytes, 0, typeAndLength, 0, 4);
                    Buffer.BlockCopy(messageLengthBytes, 0, typeAndLength, 4, 4);
                    Buffer.BlockCopy(nowInBytes, 0, typeAndLength, 8, 8);

                    _fileDescriptorStream.Write(typeAndLength, 0, LambdaTelemetryLogHeaderLength);
                    _fileDescriptorStream.Write(buffer, offset, count);
                    _fileDescriptorStream.Flush();
                }
                finally
                {
                    ArrayPool<byte>.Shared.Return(typeAndLength);
                }
            }
```